<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSDialog</class>
 <widget class="QDialog" name="OSDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OceanServer Digital Compass</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>802</width>
     <height>638</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>450</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>571</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Device</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="deviceEd">
           <property name="text">
            <string>/dev/ttyUSB0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="openBtn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Output data fields</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QListWidget" name="datalistWidget">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>101</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Azimuth</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pitch Angle</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Roll Angle</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Temperature</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Depth</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magnetic Vector Length</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3 axis Magnetic Field readings</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Acceleration Vector Length</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3 axis Acceleration Readings</string>
            </property>
            <property name="checkState">
             <enum>Unchecked</enum>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Magnetic declination</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="declinationSb">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="applyDeclBtn">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="applyDataFieldsBtn">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Output format</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="applyFormatBtn">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="outputformatCb">
           <item>
            <property name="text">
             <string>$Chhh.hPpp.pRrr.rTtt.t...*cc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>$OHPR,value1,value2,value3,...*cc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NMEA0183: $HCHDT,212.4,T*2C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Comma delimited: value1,value2,value3,...</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Offset Pitch</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Offset Roll</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Offset Heading</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QCheckBox" name="verboseCb">
           <property name="text">
            <string>Verbose</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="offsetPitchSb">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="offsetRollSb">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="offsetHeadingSb">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-90.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="dumpconfBtn">
           <property name="text">
            <string>Dump config</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QPushButton" name="helpBtn">
           <property name="text">
            <string>Help</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="applyOffsetBtn">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <widget class="QWidget" name="gaugeWidget" native="true">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>400</width>
          <height>300</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
